src/controllers/auth/addserver.js

```diff
@@ -46,9 +46,7 @@ define(["appSettings", "loading", "browser", "emby-button"], function(appSetting
         view.querySelector(".addServerForm").addEventListener("submit", onServerSubmit);
         view.querySelector(".btnCancel").addEventListener("click", goBack);
 
-        require(["autoFocuser"], function (autoFocuser) {
-            autoFocuser.autoFocus(view);
-        });
```

Comment:
@dmitrylyzo thoughts?

Reply:
The thought behind replacing this with a simpler focus is that AddServer really only has one thing you can focus on: the server address field.